byway/src/main/webapp/routepage.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Route Generator</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" 
```

Comment:
Instead of referencing this javascript library from a cdn, download the minified file (bootstrap.min.css) and store it locally in the repository. This way you are you aren't relying on a 3rd party for formatting.

Reply:
Done.

Reply:
@landondare This is litterally what CDNs are for though.

Reply:
I realize there are certain benefits from using a cdn, but I generally think it is better to have a local copy of your dependency. Especially in the context of this capstone project.

Reply:
I just used a local copy.